database/seeds/AccountsSeeder.php

```diff
@@ -10,38 +10,61 @@ class AccountsSeeder extends BaseSeeder
      */
     public function run()
     {
-        $seed_count = 50;
-
-        $types = [
-            App\CatalogEntity::class,
-            App\Fan::class
-        ];
+        $seed_count = 100;
 
         $catalogable = [
             App\Artist::class,
             App\Label::class
         ];
 
         for ($i = 1; $i <= $seed_count; $i++) {
-            $catalogable_type = $catalogable[rand(0, count($catalogable)-1)];
 
-            $type = $types[rand(0, count($types)-1)];
+            // add some randomness
+            if ([true,false][rand(0,1)]) {
```

Comment:
Not sure how I feel about this one. Why?

Reply:
The syntax kind of bothers me, maybe extracting it would help readability.

Reply:
Also, the reason for this is not clear from an outsider getting into this file. At least, I cannot see it.

Reply:
Yea, I do agree with it but it was just in the seeder for dummy data and simple randomlynpicks a true or false from the array by it's index. We can extract it to a friendlier syntax. In this context just didn't feel like it was necessary.

Reply:
I will push up a fix for this as soon as I get home.